common/src/format.rs

```diff
@@ -312,13 +334,15 @@ impl FormatSpec {
         inter: i32,
         sep: char,
         disp_digit_cnt: i32,
+        padding: PaddingStyle,
```

Comment:
I think we don't need this `padding`. It only allows whitespace or `0`, but whitespace padding doesn't add seprator.